tests/unit/components/bootstrap-datepicker-test.js

```diff
@@ -42,6 +42,32 @@ test('displays date with custom format when format is set', function(assert) {
   assert.equal(this.$().val(), '31.Dec.14');
 });
 
+test('resets date when input is cleared', function(assert) {
+  this.subject({
+    value: new Date(2014, 11, 31)
+  });
+
+  assert.ok(this.$().datepicker('getDate'), 'initial value is set');
+
+  this.$().val('');
+  this.$().trigger('input');
+
+  assert.equal(this.$().datepicker('getDate'), null, 'value is reset when input is cleared');
```

Comment:
What is this test for? It looks like testing `bootstrap-datepicker` behavior but not this addon. Shouldn't the expected value be `this.get('value')`?